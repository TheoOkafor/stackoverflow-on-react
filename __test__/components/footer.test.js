import React from 'react';
import Footer from '../../src/components/Footer';
import renderer from 'react-test-renderer';

test('Test that the Footer displays', () => {
  const component = renderer.create(
    <Footer />
  );
  expect(component).toMatchSnapshot();
});
