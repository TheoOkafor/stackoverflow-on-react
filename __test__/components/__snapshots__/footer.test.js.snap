// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test that the Footer displays 1`] = `
<footer
  className="home center"
>
  <p
    className="sitemap"
  >
    <small>
      Copyright 2018 Theo Okafor
      <a
        className="inherit"
        href="https://en.wikipedia.org/wiki/ISC_license"
      >
        (ISC License)
      </a>
    </small>
  </p>
  <p>
    <small>
      Created as part of Andela Developer Challenge
    </small>
  </p>
</footer>
`;
